### ISSUE TITLE
Non-null value 'body' was null at $.danger.github.issue.body

### ISSUE THREAD

**Issue by safall** [danger/kotlin#182]
Seems to be new issue. Even when I add body to PR, its failing. Previously it was optional and just warning is shown, now it fails

```
Run danger/kotlin@0.7.1


com.squareup.moshi.JsonDataException: Non-null value 'body' was null at $.danger.github.issue.body
	at com.squareup.moshi.kotlin.reflect.KotlinJsonAdapter.fromJson(KotlinJsonAdapter.kt:86)
	at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:137)
	at com.squareup.moshi.kotlin.reflect.KotlinJsonAdapter.fromJson(KotlinJsonAdapter.kt:83)
	at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:137)
	at com.squareup.moshi.kotlin.reflect.KotlinJsonAdapter.fromJson(KotlinJsonAdapter.kt:83)
	at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:137)
	at com.squareup.moshi.kotlin.reflect.KotlinJsonAdapter.fromJson(KotlinJsonAdapter.kt:83)
	at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:137)
	at com.squareup.moshi.JsonAdapter.fromJson(JsonAdapter.java:41)
	at systems.danger.kotlin.DangerRunner.<init>(DangerRunner.kt:125)
	at systems.danger.kotlin.DangerRunnerKt.Danger(DangerRunner.kt:238)
	at Dangerfile_df.<init>(Dangerfile.df.kts:75)
	```
Uncaught Kotlin exception: kotlin.Exception: Command kotlinc -script-templates systems.danger.kts.DangerFileScript -cp /usr/local/lib/danger/danger-kotlin.jar -script Dangerfile.df.kts /tmp/danger-dsl.json danger_out.json exited with code 768

    at kfun:systems.danger.cmd.Cmd#exec(){} (0x228518)
    at kfun:systems.danger.cmd.dangerfile.DangerFile#execute(kotlin.String;kotlin.String){} (0x22915b)
    at kfun:systems.danger.DangerKotlin#run(){} (0x227408)
    at Init_and_run_start (0x22c005)
    at __libc_start_main (0x7f0de2f50b97)
    at  (0x211029)
    at  ((nil))

Failing the build, there is 1 fail.
Request failed [403]: api.github.com/user
Response: {
  "message": "Resource not accessible by integration",
  "documentation_url": "docs.github.com/rest/reference/users#get-the-authenticated-user"
}
Request failed [403]: api.github.com/user
Response: {
  "message": "Resource not accessible by integration",
  "documentation_url": "docs.github.com/rest/reference/users#get-the-authenticated-user"
}
Uncaught Kotlin exception: kotlin.Exception: Command $(which danger) ci --process danger-kotlin --passURLForDSL  exited with code 256
    at kfun:systems.danger.cmd.Cmd#exec(){} (0x228518)
    at kfun:systems.danger.cmd.dangerjs.DangerJS#process(kotlin.String;kotlin.String;kotlin.collections.List<kotlin.String>){} (0x229764)
    at Init_and_run_start (0x22c364)
    at __libc_start_main (0x7f0d3eb59b97)
    at  (0x211029)
    at  ((nil))```

**Comment by tegorov** [comment]
Hi, @safall!
It was fixed in https://github.com/danger/kotlin/pull/175

**Comment by tegorov** [comment]
Hi, @safall!
The bug fixed in #175

**Comment by safall** [comment]
oh okay
